modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -264,6 +264,10 @@ name = HtmlUtil.escapeJS(name);
 
 			if (!validDropTarget) {
 				event.preventDefault();
+
+				if (event._event && event._event.dataTransfer) {
+					event._event.dataTransfer.dropEffect = 'none';
+				}
```

Comment:
@jbalsas, @antonio-ortega,

Thoughts on using the native browser UX instead of the Liferay.Notification()?

![native](https://user-images.githubusercontent.com/1626739/50368967-4203be00-0543-11e9-9c2c-e67969af71a2.PNG)

I say instead because when this is used it prevents the drop event, which triggers the notification.